Sources/ApodiniREST/RESTExporterConfiguration.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  RESTExporterConfiguration.swift
+//  
+//
+//  Created by Philipp Zagar on 21.05.21.
+//
+
+import Foundation
+import Apodini
+
+/// Configuration of the `RESTInterfaceExporter`
+public struct RESTExporterConfiguration {
```

Comment:
I would suggest to name this `ExporterConfiguration` and maybe make it a nested type so it is in the scope of the `REST` configuration, so you from the outside you refer to it as `REST.ExporterConfiguration`.

Reply:
I expanded this idea to all exporters, now all ExporterConfigurations are accessible via <Exporter>.ExporterConfiguration